.github/workflows/jest.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  jest:
+    name: Jest
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2.3.4
+
+    - name: install node v14
+      uses: actions/setup-node@v2.2.0
+      with:
+        node-version: 14
+        cache: 'npm'
+
+    - name: yarn install
+      run: yarn install
+
+    - name: Jest
+      run: yarn coverage
+      env:
+        TOKEN: ${{ secrets.KOREANBOTS_TOKEN }}
+        CLIENT_ID: ${{ secrets.CLIENT_ID }}
```

Comment:
Maybe these would cause the action to fail all the time. Cause these secrets would not be passed on pull_request from forked repository

Reply:
@wonderlandpark https://github.com/koreanbots/js-sdk/pull/80/checks?check_run_id=3066163325

Reply:
I'm ready to release this as a beta version when you confirm it.